docs/visual-basic/developing-apps/windows-forms/index.md

```diff
@@ -22,7 +22,7 @@ You can create Windows Forms and Windows service applications with Visual Studio
 
 ## Building Rich, Interactive User Interfaces
 
-Windows Forms is the smart-client component of the .NET Framework, a set of managed libraries that enable common application tasks such as reading and writing to the file system. Using a development environment like Visual Studio, you can create Windows Forms applications that display information, request input from users, and communicate with remote computers over a network.
+Windows Forms is the smart-client component of the .NET Framework and .NET Core (since .NET Core 3.0), a set of managed libraries that enable common application tasks such as reading and writing to the file system. Using a development environment like Visual Studio, you can create Windows Forms applications that display information, request input from users, and communicate with remote computers over a network.
```

Comment:
```suggestion
Windows Forms is the smart-client component of the .NET Framework and .NET Core (since .NET Core 3.0). It's a set of managed libraries that enable common application tasks, such as reading and writing to the file system. Using a development environment like Visual Studio, you can create Windows Forms applications that display information, request input from users, and communicate with remote computers over a network.
```